Markdown utils are broken in v2.9

## Context

Markdown utils seem to be broken in 2.9.

Raise `AttributeError: 'function' object has no attribute 'format'` instead of returning the decorated text.

* Operating System: Ubuntu 18.04.4 LTS
* Python Version: 3.8.0
* aiogram version: 2.9
* aiohttp version: 3.6.2.
* uvloop version (if installed): 0.14.0

## Expected Behavior

Markdown utility fucntions like `bold`, `hbold`, `italic`, `italics`, etc. are supposed to return decorated text.
```python
>>> aiogram.utils.markdown.hbold('test')
'<b>test</b>
```

## Current Behavior

Calling markdown utils raises an exception:
```python
AttributeError: 'function' object has no attribute 'format'`
```
This happens to a number of markdown methods: `bold`, `hbold`, `italics`, `hitalics`, etc.

## Failure Information

```python
>>> aiogram.utils.markdown.hbold('test')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/.../.virtualenvs/.../lib/python3.8/site-packages/aiogram/utils/markdown.py", line 85, in hbold
    return html_decoration.bold.format(
AttributeError: 'function' object has no attribute 'format'
```

### Steps to Reproduce

1. Import markdown utility function: `from aiogram.utils.markdown import hbold`
2. Call the function with any parameter: `hbold('test')`
3. Get the exception.

### Failure Logs

More detailed pytest log:
```python
sep = ' ', content = ('John',)

    def hbold(*content, sep=" "):
        """
        Make bold text (HTML)

        :param content:
        :param sep:
        :return:
        """
>       return html_decoration.bold.format(
            value=html_decoration.quote(_join(*content, sep=sep))
        )
E       AttributeError: 'function' object has no attribute 'format'

../../../.virtualenvs/.../lib/python3.8/site-packages/aiogram/utils/markdown.py:85: AttributeError
```

--- a/aiogram/utils/markdown.py
+++ b/aiogram/utils/markdown.py
@@ -18,7 +18,7 @@ HTML_QUOTES_MAP = {"<": "&lt;", ">": "&gt;", "&": "&amp;", '"': "&quot;"}
 _HQS = HTML_QUOTES_MAP.keys()  # HQS for HTML QUOTES SYMBOLS
 
 
-def quote_html(*content, sep=" "):
+def quote_html(*content, sep=" ") -> str:
     """
     Quote HTML symbols
 
@@ -33,7 +33,7 @@ def quote_html(*content, sep=" "):
     return html_decoration.quote(_join(*content, sep=sep))
 
 
-def escape_md(*content, sep=" "):
+def escape_md(*content, sep=" ") -> str:
     """
     Escape markdown text
 
@@ -61,7 +61,7 @@ def text(*content, sep=" "):
     return _join(*content, sep=sep)
 
 
-def bold(*content, sep=" "):
+def bold(*content, sep=" ") -> str:
     """
     Make bold text (Markdown)
 
@@ -74,7 +74,7 @@ def bold(*content, sep=" "):
     )
 
 
-def hbold(*content, sep=" "):
+def hbold(*content, sep=" ") -> str:
     """
     Make bold text (HTML)
 
@@ -87,7 +87,7 @@ def hbold(*content, sep=" "):
     )
 
 
-def italic(*content, sep=" "):
+def italic(*content, sep=" ") -> str:
     """
     Make italic text (Markdown)
 
@@ -100,7 +100,7 @@ def italic(*content, sep=" "):
     )
 
 
-def hitalic(*content, sep=" "):
+def hitalic(*content, sep=" ") -> str:
     """
     Make italic text (HTML)
 
@@ -113,7 +113,7 @@ def hitalic(*content, sep=" "):
     )
 
 
-def code(*content, sep=" "):
+def code(*content, sep=" ") -> str:
     """
     Make mono-width text (Markdown)
 
@@ -126,7 +126,7 @@ def code(*content, sep=" "):
     )
 
 
-def hcode(*content, sep=" "):
+def hcode(*content, sep=" ") -> str:
     """
     Make mono-width text (HTML)
 
@@ -139,7 +139,7 @@ def hcode(*content, sep=" "):
     )
 
 
-def pre(*content, sep="\n"):
+def pre(*content, sep="\n") -> str:
     """
     Make mono-width text block (Markdown)
 
@@ -152,7 +152,7 @@ def pre(*content, sep="\n"):
     )
 
 
-def hpre(*content, sep="\n"):
+def hpre(*content, sep="\n") -> str:
     """
     Make mono-width text block (HTML)
 
@@ -165,7 +165,7 @@ def hpre(*content, sep="\n"):
     )
 
 
-def underline(*content, sep=" "):
+def underline(*content, sep=" ") -> str:
     """
     Make underlined text (Markdown)
 
@@ -178,7 +178,7 @@ def underline(*content, sep=" "):
     )
 
 
-def hunderline(*content, sep=" "):
+def hunderline(*content, sep=" ") -> str:
     """
     Make underlined text (HTML)
 
@@ -191,7 +191,7 @@ def hunderline(*content, sep=" "):
     )
 
 
-def strikethrough(*content, sep=" "):
+def strikethrough(*content, sep=" ") -> str:
     """
     Make strikethrough text (Markdown)
 
@@ -204,7 +204,7 @@ def strikethrough(*content, sep=" "):
     )
 
 
-def hstrikethrough(*content, sep=" "):
+def hstrikethrough(*content, sep=" ") -> str:
     """
     Make strikethrough text (HTML)
 

--- a/aiogram/utils/markdown.py
+++ b/aiogram/utils/markdown.py
@@ -69,7 +69,7 @@ def bold(*content, sep=" "):
     :param sep:
     :return:
     """
-    return markdown_decoration.bold.format(
+    return markdown_decoration.bold(
         value=markdown_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -82,7 +82,7 @@ def hbold(*content, sep=" "):
     :param sep:
     :return:
     """
-    return html_decoration.bold.format(
+    return html_decoration.bold(
         value=html_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -95,7 +95,7 @@ def italic(*content, sep=" "):
     :param sep:
     :return:
     """
-    return markdown_decoration.italic.format(
+    return markdown_decoration.italic(
         value=markdown_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -108,7 +108,7 @@ def hitalic(*content, sep=" "):
     :param sep:
     :return:
     """
-    return html_decoration.italic.format(
+    return html_decoration.italic(
         value=html_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -121,7 +121,7 @@ def code(*content, sep=" "):
     :param sep:
     :return:
     """
-    return markdown_decoration.code.format(
+    return markdown_decoration.code(
         value=markdown_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -134,7 +134,7 @@ def hcode(*content, sep=" "):
     :param sep:
     :return:
     """
-    return html_decoration.code.format(
+    return html_decoration.code(
         value=html_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -147,7 +147,7 @@ def pre(*content, sep="\n"):
     :param sep:
     :return:
     """
-    return markdown_decoration.pre.format(
+    return markdown_decoration.pre(
         value=markdown_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -160,7 +160,7 @@ def hpre(*content, sep="\n"):
     :param sep:
     :return:
     """
-    return html_decoration.pre.format(
+    return html_decoration.pre(
         value=html_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -173,7 +173,7 @@ def underline(*content, sep=" "):
     :param sep:
     :return:
     """
-    return markdown_decoration.underline.format(
+    return markdown_decoration.underline(
         value=markdown_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -186,7 +186,7 @@ def hunderline(*content, sep=" "):
     :param sep:
     :return:
     """
-    return html_decoration.underline.format(
+    return html_decoration.underline(
         value=html_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -199,7 +199,7 @@ def strikethrough(*content, sep=" "):
     :param sep:
     :return:
     """
-    return markdown_decoration.strikethrough.format(
+    return markdown_decoration.strikethrough(
         value=markdown_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -212,7 +212,7 @@ def hstrikethrough(*content, sep=" "):
     :param sep:
     :return:
     """
-    return html_decoration.strikethrough.format(
+    return html_decoration.strikethrough(
         value=html_decoration.quote(_join(*content, sep=sep))
     )
 
@@ -225,7 +225,7 @@ def link(title: str, url: str) -> str:
     :param url:
     :return:
     """
-    return markdown_decoration.link.format(value=markdown_decoration.quote(title), link=url)
+    return markdown_decoration.link(value=markdown_decoration.quote(title), link=url)
 
 
 def hlink(title: str, url: str) -> str:
@@ -236,7 +236,7 @@ def hlink(title: str, url: str) -> str:
     :param url:
     :return:
     """
-    return html_decoration.link.format(value=html_decoration.quote(title), link=url)
+    return html_decoration.link(value=html_decoration.quote(title), link=url)
 
 
 def hide_link(url: str) -> str:

